rpi4-ws: Update readme and scripts

3mkusiak opened the pull request:
Remove outdated URLs and fixe a logic in build.sh to build cba applications properly.

3mkusiak committed:
rpi4-ws: Update readme and scripts

The following commit removes outdated URLs and fixes a logic in build.sh
to build cba applications properly.

Signed-off-by: Mateusz Kusiak <mateusz.kusiak@3mdeb.com>

m-iwanicki commented on rpi4-ws/README.md:
````diff
@@ -8,10 +8,12 @@ build virtual TEEs.
 cd crosscon-demos
 ```
 
+_Note: Change link for `https` if your're not the member of organization._
````
You should maybe mention exact link (<https://github.com/crosscon/crosscon-demos-uc12.git>)?
Not only that but some submodules will fail to clone anyway e.g.:

```
fatal: clone of 'git@github.com:crosscon/bitcoin-wallet.git' into submodule path '/git/crosscon-demos/bitcoin-wallet' failed
Failed to clone 'bitcoin-wallet' a second time, aborting
```

3mkusiak replied:
Updated: https://github.com/crosscon/crosscon-demos-uc12/compare/fa03d84bfb3541f828a79475e82d42b465c890cf..01fb08468c197fa3b418d9b634c14fdf14ccfd77

I believe it's redundant for me to give full URL if users need to update submodules.

3mkusiak replied:
We discussed offline to replace all url with HTTP, working on that.

3mkusiak committed:
Use https instead of ssh

Update submodules to use https and update manual in README. This is so
folks outside of the organization could access these.

Signed-off-by: Mateusz Kusiak <mateusz.kusiak@3mdeb.com>

3mkusiak commented:
@DaniilKl need ya to merge this probably
